app/2.0/docs/tools/create-app-polymer-cli.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Create an application project with the Polymer CLI
+---
+
+<!-- toc -->
+
+Polymer CLI supports initializing a project folder with one of several application templates.  The 
+CLI comes with a `basic` template, which is the most basic starting point for a Polymer-based 
+application, as well as others that introduce more complex layout and application patterns.
+
+This chapter teaches you more about `basic` app projects.  See [Polymer App Toolbox 
+templates](../../toolbox/templates) for more details on other templates.
+
+## App project architecture {#app-architecture}
+
+Polymer CLI is designed for apps that follow the [app shell 
```

Comment:
New user 🎩: Wait, Is it designed *only* for apps that follow the app shell architecture? If I'm not following that, i can't use the CLI?